spec/till_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'till'
+require 'timecop'
+
+describe Till do
+
+  let(:till) { described_class.new( { shopName: "The Coffee Connection",
+                                      address: "123 Lakeside Way",
+                                      phone: "16503600708",
+                                      prices: { "Cafe Latte"=> 4.75, "Flat White"=> 4.75 }
+                                     } ) }
+
+  let(:order) { double(:order, customers: ["Jane", "John"], 
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.